packages/graphback-runtime-knex/src/PgKnexDBDataProvider.ts

```diff
@@ -20,7 +20,7 @@ export class PgKnexDBDataProvider<Type = any> extends KnexDBDataProvider<Type> {
     const { data: createData } = getDatabaseArguments(this.tableMap, data);
 
     //tslint:disable-next-line: await-promise
-    const dbResult = await this.db(this.tableName).insert(createData).returning(context.graphback.options.selectedFields);
+    const dbResult = await this.db(this.tableName).insert(createData).returning(this.getSelectedFields(context));
```

Comment:
We can actually delete this file (no longer used, moved to KnexDBDataProvider)

Reply:
You mean this file or method?

Reply:
File/class.

Reply:
It's deprecated: https://graphback.dev/docs/next/releases#deprecated

But we can probably just remove it since its pre-v1.

Reply:
I prefer if we have a separate issue "technical debt" to remove it.

Reply:
Absolutely.